lib/pages/contacts_tab/contacts_tab.dart

```diff
@@ -45,6 +58,17 @@ class ContactsTabController extends State<ContactsTab>
 
     _listenFocusTextEditing();
     super.initState();
+    WidgetsBinding.instance.addObserver(this);
+  }
+
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) {
+    if (state == AppLifecycleState.resumed) {
+      initialFetchContacts(
```

Comment:
Why we need to fetch contacts?

Reply:
Side effect

https://github.com/user-attachments/assets/09dab8b3-ebd5-4d76-9dba-e5ac4b9b4d1d

Reply:
Not a side effect.